neo/app/views/index.scala.html

```diff
@@ -1,15 +1,31 @@
 @(title: String) @main(title){
 <div xmlns="http://www.w3.org/1999/html" ng-controller="AppCtrl">
 
-  <map lat="39.5" lng="-96.35" zoom="4"></map>
+  <div class="map-group">
 
-  <search-bar></search-bar>
 
-  <time-series id="chart"></time-series>
+    <map lat="39.5" lng="-96.35" zoom="4"></map>
 
+    <div id="logo">
+      <img src ="https://asterixdb.apache.org/img/asterixdb_tm.png" id="logo_img">
```

Comment:
let's remove the logo for now. we haven't decided which one to use. it's fine to keep the `div` here.